clients/unshaded/src/main/java/tachyon/client/WriteType.java

```diff
@@ -16,32 +16,36 @@
 package tachyon.client;
 
 /**
- * Different write types for a TachyonFile.
+ * Convenience modes for commonly used write types for a TachyonFile.
  *
- * As of 0.8, replaced by {@link TachyonStorageType} and {@link UnderStorageType}
+ * For finer grained control over data storage, advanced users may specify
+ * {@link tachyon.client.TachyonStorageType} and {@link tachyon.client.UnderStorageType}.
  */
-@Deprecated
 public enum WriteType {
   /**
-   * Write the file and must cache it.
+   * Write the file and must cache it. No data will be persisted to the under storage.
```

Comment:
Which tier will the data be written into? The description hints the data pinned in memory?